src/tty/windows.rs

```diff
@@ -220,13 +231,21 @@ impl RawReader for ConsoleRawReader {
                 };
                 let c = rc?;
                 if meta {
-                    return Ok(KeyPress::Meta(c));
+                    let mut key = keys::char_to_key_press(c);
```

Comment:
You can factorize duplicated lines 234 and 242 by moving it at line 233.

Reply:
understood.